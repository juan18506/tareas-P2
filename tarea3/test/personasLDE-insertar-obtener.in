crearListaPersonas

crearAgenda
crearPersona 1 18 Persona1
crearFecha 18/4/2023
crearEvento 1 Descripcion uno
agregarEventoPersona
insertarPersonaInicioListaPersonas

crearAgenda
crearPersona 2 20 Persona 2
insertarPersonaInicioListaPersonas

crearAgenda
crearPersona 3 25 Persona3
insertarPersonaFinalListaPersonas

crearAgenda
crearPersona 4 30 Persona4
crearFecha 19/4/2023
crearEvento 1 Descripcion uno
agregarEventoPersona
crearFecha 20/4/2023
crearEvento 2 Descripcion dos
agregarEventoPersona
insertarPersonaFinalListaPersonas

imprimirCantidadListaPersonas
imprimirListaPersonas

# imprime Persona 2
imprimirInicioListaPersonas

# imprime Persona 4
imprimirFinalListaPersonas

eliminarInicioListaPersonas

# imprime Persona 1
imprimirInicioListaPersonas

concatenarConVacioListaPersonas

# imprime Persona 4
imprimirFinalListaPersonas

eliminarFinalListaPersonas

# imprime Persona 3
imprimirFinalListaPersonas

eliminarInicioListaPersonas

# imprime la misma persona
imprimirInicioListaPersonas
imprimirFinalListaPersonas

liberarListaPersonas

Fin
