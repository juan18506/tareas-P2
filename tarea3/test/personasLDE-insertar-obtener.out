1>2>3>4>5>6>7>8>9>10>11>12>13>14>15>16>17>18>19>20>21>22>23>La cantidad de personas en la lista es: 4.
24>Persona 2:  Persona 2, 20 años
Persona 1:  Persona1, 18 años
Evento 1:  Descripcion uno
Fecha: 18/4/2023
Persona 3:  Persona3, 25 años
Persona 4:  Persona4, 30 años
Evento 1:  Descripcion uno
Fecha: 19/4/2023
Evento 2:  Descripcion dos
Fecha: 20/4/2023
25>#  imprime Persona 2.
26>Persona 2:  Persona 2, 20 años
27>#  imprime Persona 4.
28>Persona 4:  Persona4, 30 años
Evento 1:  Descripcion uno
Fecha: 19/4/2023
Evento 2:  Descripcion dos
Fecha: 20/4/2023
29>30>#  imprime Persona 1.
31>Persona 1:  Persona1, 18 años
Evento 1:  Descripcion uno
Fecha: 18/4/2023
32>33>#  imprime Persona 4.
34>Persona 4:  Persona4, 30 años
Evento 1:  Descripcion uno
Fecha: 19/4/2023
Evento 2:  Descripcion dos
Fecha: 20/4/2023
35>36>#  imprime Persona 3.
37>Persona 3:  Persona3, 25 años
38>39>#  imprime la misma persona.
40>Persona 3:  Persona3, 25 años
41>Persona 3:  Persona3, 25 años
42>43>Fin.
